src/main/java/uk/gov/register/views/representations/YamlWriter.java

```diff
@@ -26,7 +26,7 @@ public YamlWriter() {
     }
 
     @Override
-    public void writeTo(Object object, Class<?> type, Type genericType, Annotation[] annotations, MediaType mediaType, MultivaluedMap<String, Object> httpHeaders, OutputStream entityStream) throws IOException, WebApplicationException {
+    public void writeTo(final Object object, final Class<?> type, final Type genericType, final Annotation[] annotations, final MediaType mediaType, final MultivaluedMap<String, Object> httpHeaders, final OutputStream entityStream) throws IOException, WebApplicationException {
```

Comment:
Just as a general note (and I know this is from the auto-formatter), I don't think we should be including files where the actual file hasn't changed in some meaningful way related to the problem the PR is trying to address.